### PR TITLE
Document websocket support and fine tuning. 

### PR DESCRIPTION
**Pull request by christian-posta**
Related to https://github.com/solo-io/gloo/pull/1784 and fixes https://github.com/solo-io/gloo/issues/2143

### CODE REVIEW

**Comment by christian-posta** [comment]
cc @kdelorey

**Comment by kdelorey** [comment]
I'm not sure how to render the docs, but the content lgtm 👍

File: docs/content/gloo_routing/websockets.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  useProxyProto: false
+{{< /highlight >}}
+
+Note the `httpGateway` configuration above with settings for the `httpConnectionManager`. 
+
+We can also configure whether websocket is enabled per route as we see in the next section.
+
+## Configuring websocket per route
+
+If you need more fine-grained control over how websockets is configured, you can use the {{< protobuf name="gateway.solo.io.Gateway" display="Route">}} configuration in a {{< protobuf name="gateway.solo.io.Gateway" display="VirtualService">}}. For example, if you have websockets disabled for you listener (as seen in previous section), you can enable it per route with the following:
```
Comment (kdorosh) [inline docs/content/gloo_routing/websockets.md]:
```suggestion
If you need more fine-grained control over how websockets are configured, you can use the {{< protobuf name="gateway.solo.io.Gateway" display="Route">}} configuration in a {{< protobuf name="gateway.solo.io.Gateway" display="VirtualService">}}. For example, if you have websockets disabled for you listener (as seen in previous section), you can enable it per route with the following:
```

Comment (kdorosh) [reply docs/content/gloo_routing/websockets.md]: Also, looks like we need to update the protobuf linked in each case from Gateway to the appropriate resource

Comment (christian-posta) [reply docs/content/gloo_routing/websockets.md]: K made changes, take a look

File: docs/content/gloo_routing/websockets.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  useProxyProto: false
+{{< /highlight >}}
+
+Note the `httpGateway` configuration above with settings for the `httpConnectionManager`. 
+
+We can also configure whether websocket is enabled per route as we see in the next section.
+
+## Configuring websocket per route
+
+If you need more fine-grained control configuring websockets, you can use the {{< protobuf name="gateway.solo.io.Route" display="Route">}} configuration in a {{< protobuf name="gateway.solo.io.VirtualService" display="VirtualService">}}. For example, if you have websockets disabled for you listener (as seen in previous section), you can enable it per route with the following:
```
Comment (kdorosh) [inline docs/content/gloo_routing/websockets.md]: grammar,
```suggestion
If you need more fine-grained control configuring websockets, you can use the {{< protobuf name="gateway.solo.io.Route" display="Route">}} configuration in a {{< protobuf name="gateway.solo.io.VirtualService" display="VirtualService">}}. For example, if you have websockets disabled for your listener (as seen in previous section), you can enable it per route with the following:
```

**Pull request merged**